packages/sit-onyx/src/components/OnyxBasicDialog/OnyxBasicDialog.ct.tsx

```diff
@@ -23,9 +23,29 @@ test("should render", async ({ mount, makeAxeBuilder, page }) => {
   await expect(page).toHaveScreenshot("default.png");
 });
 
-test("should render in modal", async ({ mount, makeAxeBuilder, page }) => {
+const BACKDROPS = ["light", "intense"] as const;
+
+for (const backdrop of BACKDROPS) {
+  test(`should render in modal with ${backdrop} backdrop`, async ({
+    mount,
+    makeAxeBuilder,
+    page,
+  }) => {
+    await mount(
+      <OnyxBasicDialog label="Label" open modal backdrop={backdrop}>
+        Content
+      </OnyxBasicDialog>,
+    );
+
+    const accessibilityScanResults = await makeAxeBuilder().analyze();
+    expect(accessibilityScanResults.violations).toEqual([]);
+
+    await expect(page).toHaveScreenshot(`modal-${backdrop}.png`);
+  });
+}
+test("should render in modal (intense)", async ({ mount, makeAxeBuilder, page }) => {
```

Comment:
Isn't this test duplicated since we are already have the for loop at the top that executes the test for all intensities?